### PR TITLE
feat(collector-form-store): connect store and version form component

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Variable name changed from `autoUpdate` to `autoUpgrade`
Add provider state to collector-form-store

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep1.vue [hunk 1]
```diff
@@ -101,7 +98,7 @@ const getPlugins = async (): Promise<CollectorPluginModel[]> => {
         const params = {
             service_type: 'inventory.Collector',
             repository_id: state.selectedRepository === 'all' ? '' : state.selectedRepository,
-            provider: collectorPageState.selectedProvider === 'all' ? '' : collectorPageState.selectedProvider,
+            provider: collectorFormState.provider === 'all' ? '' : collectorFormState.provider,
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep1.vue:101]: issue: I think it must be `undefined` when all providers are selected.

**Review by WANZARGEN — COMMENTED** [review]
Thanks for refactoring! :)
Overally, the codes are very readable! I love it.
I think there is an issue about 'provider' state, so I left a comment. Please check it. :)

**Review by yuda110 — APPROVED** [review]
LGTM! anyway there's conflict...😅

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**